src/plone/restapi/tests/http-examples/linkintegrity_post.resp

```diff
@@ -0,0 +1,20 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+    {
+        "@id": "http://localhost:50989/plone/doc-2",
+        "@type": "Document",
+        "breaches": [
+            {
+                "@id": "http://localhost:50989/plone/doc-1",
+                "accessible": 1,
```

Comment:
What does "accessibility" means here?

Reply:
probably better named "allowed"? (if get it right)

Reply:
removed that info because right now we don't need it..if needed i can take it back and maybe rename it